### PR TITLE
Add EnableApp, StartApp and DisableApp

### PR DESCRIPTION
**Pull request by guillemcordoba**
@zo-el if you'd like I also can remove all the tests with the deprecated functions, and replace them with the newer functions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ See [docs/API.md](docs/API.md)
 # Holochain Compatibility
 
 This version of `holochain-conductor-api` is currently working with
-- `holochain/holochain` at commit: [c6e501eceae06f381c80ed50ea7bb971f8d1edcc](https://github.com/holochain/holochain/commit/c6e501eceae06f381c80ed50ea7bb971f8d1edcc)
+- `holochain/holochain` at commit: [1eb5b70f64b2188e6ef0c876367838b8a6a2015d](https://github.com/holochain/holochain/commit/1eb5b70f64b2188e6ef0c876367838b8a6a2015d)
 - hdk version 0.0.100 from crates.io
```
Comment (zo-el) [inline README.md]: can we update this too?

Also, it would be useful to add a date of the holochain commit just for reference here

**Comment by zo-el** [comment]
> @zo-el if you'd like I also can remove all the tests with the deprecated functions, and replace them with the newer functions.

@guillemcordoba - That would be nice

File: README.md (hunk 1 above)
Comment (guillemcordoba) [reply README.md]: Yeap yeap

**Review by zo-el — APPROVED** [review]
nice work

**Review by JettTech — COMMENTED** [review]
Thanks for pushing this forward @guillemcordoba! :tada:

**Pull request merged**